database/index.js

```diff
@@ -23,4 +50,63 @@ const getWatchInfo = (watchId, callback) => {
   });
 };
 
-module.exports = { getWatchInfo };
+const updateWatchInfo = (watchId, watchName, callback) => {
+  const queryStringWatchInfo = 
+  `UPDATE watches SET 
+    wid = "${watchId}", 
```

Comment:
You'll actually receive watchId and watchName as strings from your request body so the quotation marks aren't necessary.